deploy/olm-catalog/kubeflow/kubeflow.package.yaml

```diff
@@ -1,5 +1,5 @@
 channels:
-- currentCSV: kubeflow.v0.1.0
+- currentCSV: kubeflow.v1.0.1
```

Comment:
What will correspond to version 1.0?

Reply:
thinking whether we should update v1.0-branch. If it is not possible to update that, then we will have to spell out in the document.

Reply:
Lets change the version to 1.0.0 

otherwise
/lgtm

Reply:
Done. Thanks @animeshsingh